Source/ThirdParty/libwebrtc/Source/webrtc/modules/video_coding/codecs/h264/h264_encoder_impl.cc

```diff
@@ -34,10 +34,17 @@
 #include "system_wrappers/include/metrics.h"
 #include "third_party/libyuv/include/libyuv/convert.h"
 #include "third_party/libyuv/include/libyuv/scale.h"
+#if defined(WEBKIT_LIBWEBRTC_OPENH264_ENCODER) && WEBKIT_LIBWEBRTC_OPENH264_ENCODER
```

Comment:
Can we add `defined(WEBRTC_WEBKIT_BUILD)` as well, this will make it clearer for me next time I resend libwebrtc.

Reply:
Done.